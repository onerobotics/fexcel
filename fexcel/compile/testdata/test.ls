/PROG  TEST
/ATTR
COMMENT		= "";
DEFAULT_GROUP	= 1,*,*,*,*;
/APPL
AUTO_SINGULARITY_HEADER;
  ENABLE_SINGULARITY_AVOIDANCE   : TRUE;
/MN
 : ! this is a valid file ;
 : R{one}=R{two}+R{three} ;
 : J PR{home} ${HOME_SPEED}% CNT${HOME_CNT} ;
 : PR{lpos}=LPOS ;
 : PR{jpos}=JPOS ;
/POS
/END
