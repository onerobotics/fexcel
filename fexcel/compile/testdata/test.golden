/PROG  TEST
/ATTR
COMMENT		= "";
DEFAULT_GROUP	= 1,*,*,*,*;
/APPL
AUTO_SINGULARITY_HEADER;
  ENABLE_SINGULARITY_AVOIDANCE   : TRUE;
/MN
 : ! this is a valid file ;
 : R[1:one]=R[2:two]+R[3:three] ;
 : J PR[4:home] 100% CNT0 ;
 : PR[5:lpos]=LPOS ;
 : PR[6:jpos]=JPOS ;
/POS
/END
